scripts/docker-manylinux.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
+
+BUILD_DIR="/work"
+
+docker run \
+        -w /work/py \
+        -v `pwd`:/work \
+        quay.io/pypa/manylinux1_${BUILD_ARCH} \
+        sh manylinux.sh
+
+# Fix permissions for shared directories
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+sudo chown -R ${USER_ID}:${GROUP_ID} target/
```

Comment:
Can you check if we can remove this? It may only be a relic in Relay, since in current symbolic Travis builds we do not need to do this. 

Note that for release builds we do not cache the target folder in Relay.